docs/start.rst

```diff
@@ -3,7 +3,44 @@
 Getting Started
 ================
 
-Overview
-########
+This library is an end-to-end audio/text aligner. It is meant to be used
+together with the ReadAlong-Web-Component to interactively visualize the
+alignment.
 
-...more to come...
+Background
+----------
+
+The concept is a web application with a series of stages of processing,
+which ultimately leads to a time-aligned audiobook, i.e., a package of:
+
+-  SMIL file describing time alignments
+-  XML file describing text
+-  Audio file (WAV or MP3)
+-  HTML file describing the web component
+
+Which can be loaded using the `read-along web
+component <https://github.com/roedoejet/ReadAlong-Web-Component>`__.
+
+A book is generated as a standalone HTML page by default, but can
+optionally be generated as an ePub file.
+
+Required knowledge
+------------------
+
+-  Command line interface (CLI)
```

Comment:
This is great, but I would like to see links for each of these. Like maybe a link to https://developer.mozilla.org/en-US/docs/Learn/Common_questions/set_up_a_local_testing_server with the "spinning up a server" item. Also, I think verbs are better than nouns for these lists - ie what exactly do we expect them to do with "Plain text file/xml/smil"? Maybe change to "Edit and manipulate plain text, XML (linked), and SMIL (linked) files using a text or code editor" for example